app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,54 @@
+# encoding: utf-8
+
+class AvatarUploader < CarrierWave::Uploader::Base
+
+  # Include RMagick or MiniMagick support:
+  # include CarrierWave::RMagick
+  include CarrierWave::MiniMagick
+
+  # if Rails.env.production? || Rails.env.development?
+    storage :fog
+  # else
+  #   storage :file
+  # end
+
+  # Override the directory where uploaded files will be stored.
+  # This is a sensible default for uploaders that are meant to be mounted:
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  # Provide a default URL as a default if there hasn't been a file uploaded:
+  def default_url
+    [version_name, "default.png"].compact.join('_')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.